src/cata_utility.cpp

```diff
@@ -282,3 +282,8 @@ double convert_weight( int weight )
     }
     return ret;
 }
+
+double temp_to_celsius( float fahrenheit )
+{
+    return ( ( fahrenheit - 32 ) * 5 / 9 );
```

Comment:
This is going to truncate like crazy with those bare int literals, I'll add some .0s in the merge.